docs/source/index.rst

```diff
@@ -125,6 +125,7 @@ Documentation
    :caption: User Guides
 
    examples/docker-containers
+   examples/ports
```

Comment:
I think you forgot to add the ports page source to git :)

Reply:
Thanks for letting me know! Now added ; )